crates/db_schema/src/source/languages.rs

```diff
@@ -0,0 +1,10 @@
+use crate::{schema::user_languages, LocalUserId, PrimaryLanguageTag};
+use serde::Serialize;
+
+#[derive(Clone, Queryable, Identifiable, PartialEq, Debug, Serialize)]
+#[table_name = "user_languages"]
+pub struct UserLanguages {
+  pub id: i32,
+  pub local_user_id: LocalUserId,
+  pub language: PrimaryLanguageTag,
```

Comment:
The actual column here is a string, that's why its failing.

Reply:
Changing it to string doesnt help, the errors are coming from local_user_view.rs, because I dont know how to join the UserLanguage rows into a per-user Vec.

Reply:
I wiped the commits for the user_languages table, because I would rather get the main funcionality working first. Later we can see about improving the tables etc.